angr/analyses/propagator/propagator.py

```diff
@@ -271,13 +280,13 @@ def _pre_job_handling(self, job):
     def _initial_abstract_state(self, node):
         if isinstance(node, ailment.Block):
             # AIL
-            state = PropagatorAILState(arch=self.project.arch)
+            state = PropagatorAILState(arch=self.project.arch,only_consts=self._only_consts)
```

Comment:
I was surprised that this passed the lint and thought that something is wrong with CI for a moment. I noticed that we do seem to have this disabled in the pylintrc. Is this something that we want to remove from that so that this kind of formatting is caught?

Reply:
Are you talking about the missing space after the comma?

Reply:
Do you need me to update that?

Reply:
@Itfish yeah, that jumped out at me when glancing at the diff

Reply:
@mahaloz please update it.

Reply:
Also, due to my recent commits, your branch now has conflicts to the master branch. Can you resolve them first?

Reply:
@ltfish conflicts resolved. Line fixed.